templates/usr/local/sbin/simp_generate_types.epp

```diff
@@ -0,0 +1,104 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
This looks like a plain file, which is ok to leave as a template

Reply:
That's mainly for working around masterless scenarios.